src/App.tsx

```diff
@@ -1,10 +1,43 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback, useEffect, useMemo, useState,
+} from 'react';
 import { UserWarning } from './UserWarning';
+import { HeaderTodoApp } from './components/HeaderTodoApp';
+import { MainTodoApp } from './components/MainTodoApp';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { FooterTodoApp } from './components/FooterTodoApp';
+import { Filters } from './types/Filters';
```

Comment:
Add `Filters` file to the PR
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/91411692/844b3656-b8ef-4b31-b689-056277db0fba)